deployment/ccip/changeset/cs_initial_add_chain.go

```diff
@@ -59,6 +60,16 @@ type CCIPOCRParams struct {
 	ExecuteOffChainConfig pluginconfig.ExecuteOffchainConfig
 }
 
+// Override overrides non-empty dst CCIPOCRParams attributes with non-empty src CCIPOCRParams attributes values
```

Comment:
Idea is we will use DefaultOCRParams and only mention the non-nil param values to override from chainlink-deployments